stages/org.osbuild.firewall

```diff
@@ -92,6 +119,36 @@ def main(tree, options):
                        list(map(lambda x: f"--remove-service={x}", disabled_services)),
                        check=True)
 
+    for zone_item in zones:
+        # specifying an empty zone flag results in the source being applied to
+        # the default zone
+        zone_name = zone_item['name']
+        # check that the given zone exists, if not create it
+        if zone_name != "":
+            res = subprocess.check_output(["chroot",
+                                           tree,
+                                           "firewall-offline-cmd",
+                                           "--get-zones"])
+            if res.find(zone_name.encode()) < 0:
```

Comment:
This might easily fail if one zone is a sub-string of another, right? Can't we do `--info-zone=<zone>` instead, and check the return value? (assuming it fails if the zone does not exist)